fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/api/LoansApiResource.java

```diff
@@ -639,6 +640,12 @@ public String retrieveLoan(@PathParam("loanId") @Parameter(description = "loanId
                 linkedAccount = this.accountAssociationsReadPlatformService.retriveLoanLinkedAssociation(loanId);
             }
 
+            if (associationParameters.contains("collection")) {
```

Comment:
I know the existing codebase doesn't really use constants but rather hardcoded values; can we make sure to extract the "collection" magic string to somewhere and just reference it? 
Would be a shame if in the future some functionality breaks because we change it here but not the other place(s).